packages/ref-doc/src/experimental.ts

```diff
@@ -0,0 +1,19 @@
+import { compile, joinPaths, NodeHost } from "@cadl-lang/compiler";
+import { mkdir, writeFile } from "fs/promises";
+import { renderToDocusaurusMarkdown } from "./emitters/docusaurus.js";
+import { extractRefDocs } from "./extractor.js";
+
+/**
+ * @experimental this is for experimental and is for internal use only. Breaking change to this API can happen at anytime.
```

Comment:
Good call! 

Curious, does IDE do anything interesting with this or is just that you'll see it in hover/signature-help/intellisense with the docs?

Reply:
I don’t think it does. I was thinking of using deprecated to have it show crossed but that also felt a little wrong.

the package is not published anyway so don’t think it’s a big deal